src/ui/styles/app.css

```diff
@@ -0,0 +1,2 @@
+@import 'todomvc-app-css/index.css';
```

Comment:
Sure. I just updated this PR. Let me know if this is what you were thinking about.